src/css/cs/items/_kitchensink.scss

```diff
@@ -376,6 +376,19 @@
     }
   }
 
+  // Style the text that appears in the HTMLArea on UC_STATUS_MSG_SBF
+  .uc-status-message {
+    border: 0;
+    box-shadow: none;
+    font-size: 16px;
+    font-weight: bold;
+    margin: 0;
+    outline: none;
```

Comment:
@marcpechaitis not a blocker but would suggest seeing whether we can disabe this to make it even more accessible (this allows people to see where the focus is)